packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -131,6 +133,16 @@ class PaginatedDataTable extends StatefulWidget {
   /// See [DataTable.onSelectAll].
   final ValueSetter<bool> onSelectAll;
 
+  /// The height of each row (excluding the row that contains column headings).
+  ///
+  /// This value is optional and defaults to 48.0 if not specified.
+  final double dataRowHeight;
+
+  /// The height of the heading row.
+  ///
+  /// This value is optional and defaults to 56.0 if not specified.
```

Comment:
I posted above about this line (and the above for `dataRowHeight`) being confusing, and the same thing applies here.

However, I also want to point out here that this class doesn't default anything (`DataTable` does).  So I think maybe you should just remove this line about the default here.

Reply:
Took the approach mentioned in https://github.com/flutter/flutter/pull/33628#discussion_r292634315